cron/monthly/user-report.js

```diff
@@ -451,6 +452,7 @@ const computeStats = async (collectives, currency = 'USD') => {
   stats.expensesBreakdownString = `${Object.keys(categories).length > 3 ? ', mostly in' : ' in'} ${formatArrayToString(
     ar,
   )}`;
+  console.log(`>>> Stats: ${JSON.stringify(stats, null, 2)}`);
```

Comment:
Remove console.log ?

Reply:
I believe this `console.log` may be really helpful to check the logs after running these scripts. in this case it'll show the consolidated stats per user. I think it worths keeping this console.log as we already have other logs in this report as well. But let me know if you think it's too much.

Reply:
afaik the same way we can log the app but instead of using `--app` we'd use the scheduler job id. example: `heroku logs -d scheduler.5255`